packages/terra-tabs/README.md

```diff
@@ -0,0 +1,27 @@
+# Terra Tabs
+
+
+[![NPM version](http://img.shields.io/npm/v/terra-tabs.svg)](https://www.npmjs.org/package/terra-tabs)
+[![Build Status](https://travis-ci.org/cerner/terra-core.svg?branch=master)](https://travis-ci.org/cerner/terra-core)
+
+Tabs are containers used to organize content. They allow for quick switching between groups of contextually related content. Content is divided into different containers and each container is viewable one at a time. The user can switch between containers by selecting the corresponding tab control.
+
+Tabs are meant for organizing content that doesn’t need to be compared or accessed simultaneously or if the user needs quick access through a group of contextually related content. Tabs are not intended for navigation.
```

Comment:
This is a little confusing.
> Tabs are meant for organizing content that doesn’t need to be compared or accessed simultaneously or if the user needs quick access through a group of contextually related content.

Did you mean something like?

Tabs are meant to organize content as to provide quick access to a group of contextually related content that does not need to be compared or accessed simultaneously.

Reply:
Honestly, I copied this from the description in zeplin. I can reword it.

Reply:
Updated: https://github.com/cerner/terra-core/pull/1054/commits/4fcabc2a3ba1ef3adbb0168c8f31be5c1caba8a3